steps/dexprotector/1.0.1/step.yml

```diff
@@ -0,0 +1,191 @@
+    title: Certificate location
+- opts:
+    category: Android specific options
+    is_expand: true
+    is_sensitive: true
+    summary: 'For `google` or `amazon` sign mode only. Specifies the target certificate''s
+      SHA256 certificate fingerprint. The value of this fingerprint can be found:
+      Amazon: Amazon Developer Console, then go to settings of your application: Application
+      -> Binary Files -> Appstore Certificate Hashes -> SHA-256 -> Hexadecimal. Google:
+      Google Play Console, then go to settings of your application: App Signing ->
+      App signing certificate -> SHA-256 certificate fingerprint'
+    title: SHA256 certificate fingerprint of the target certificate
+  sha256_certificate_fingerprint: null
+- keystore: $BITRISEIO_ANDROID_KEYSTORE_URL
+  opts:
+    category: Android specific options
+    is_expand: true
+    summary: Path to keystore. For `release` sign mode only. Specifies the path to
+      the keystore containing the relevant signing key. This option can also be specified
+      in the configuration file.
+    title: Path to keystore
+- opts:
+    category: Android specific options
+    is_expand: true
+    is_sensitive: true
+    summary: Password for keystore integrity. For `release` sign mode only. Specifies
+      the password to the keystore containing the relevant signing key. This option
+      can also be specified in the configuration file.
+    title: Password for keystore integrity
+  storepass: $BITRISEIO_ANDROID_KEYSTORE_PASSWORD
+- keyalias: $BITRISEIO_ANDROID_KEYSTORE_ALIAS
+  opts:
+    category: Android specific options
+    is_expand: true
+    summary: Key alias. For `release` sign mode only. Specifies the alias string for
+      the relevant signing key in your keystore. This option can also be specified
+      in the configuration file.
+    title: Key alias
+- keypass: $BITRISEIO_ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD
+  opts:
+    category: Android specific options
+    is_expand: true
+    is_sensitive: true
+    summary: Password for the private key (if different). For `release` sign mode
+      only. Specifies the password for the signing key. This option can also be specified
+      in the configuration file.
+    title: Password for the private key (if different)
+- opts:
+    category: Android specific options
+    is_expand: true
+    summary: ProGuard mapping file path. Specifies the path to ProGuard's ‘mappings.txt’
+      file. If you use ProGuard for name obfuscation, it is necessary to provide the
+      path to ProGuard's map file so that DexProtector can locate classes for encryption.
+      This option can also be specified in the configuration file.
+    title: ProGuard mapping file path
+  proguard_map_file: $BITRISE_MAPPING_PATH
+- opts:
+    category: Debug
+    title: Print the verbose protection build log to stdout. This option can also
+      be specified in the configuration file.
+    value_options:
```

Comment:
These fields must be `string` types.

Step validation failed in the CI:
```
.....
- (2/4) Running: Ensure all value_option is string type
 > Failed, error:
   - type(bool) in value_options with value: true at input: Print the verbose protection build log to stdout. This option can also be specified in the configuration file.
   - type(bool) in value_options with value: false at input: Print the verbose protection build log to stdout. This option can also be specified in the configuration file.

.....
```

Reply:
In [original file](https://github.com/dexprotector/bitrise-step-dexprotector/blob/main/step.yml) it is `yes` and `no` options. 
Maybe it generated by `bitrise run share-this-step` ?

Reply:
take a look at the enclosing quote in the original file. It is required to wrap `yes / no` or `true / false` in a quote otherwise it won't get parsed correctly (hence the build failure). So it's a string list not a boolean list. Please put either:
```
- 'yes'
- 'no'
```

or 

```
- 'true'
- 'false
```

Reply:
@tadam313 ok, thanks